tests/test_bills.py

```diff
@@ -86,7 +85,7 @@ def test_bill_manager(bill,
     '''
     bill_info = {
         'classification': [bill_type],
-        'extras': {'restrict_view': restrict_view},
+        'restrict_view': restrict_view,
```

Comment:
This is [an attribute of the Councilmatic model now](https://github.com/datamade/django-councilmatic/blob/85e518ab6fb912a2bfa574e4e53abd82a2d1f412/councilmatic_core/models.py#L386).